### PR TITLE
Add code to handle collectiontype in form describer

### PR DESCRIPTION
**Pull request by matrix818181**
Currently, `FormModelDescribe `doesn't handle `CollectionType`.

So I propose a quick fix to enable it.

### CODE REVIEW

**Comment by dstmichel** [comment]
Well done, good idea. +1

File: .travis.yml [hunk 1]
```diff
@@ -22,14 +22,9 @@ matrix:
...
     - php: 7.0
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: 7.1
-      env: SF_4="true"
 
 before_install:
     - phpenv config-rm xdebug.ini
-    - if [ "$SF_4" == "true" ]; then composer config minimum-stability dev; fi
-    - if [ "$SF_4" == "true" ]; then composer require symfony/symfony:4.0.* --dev --no-update; fi
-    - if [ "$SF_4" == "true" ]; then composer require friendsofsymfony/rest-bundle:dev-master --dev --no-update; fi
```
Comment (GuilhemN) [inline .travis.yml]: Please revert this change

Comment (matrix818181) [reply .travis.yml]: But the Travis CI will failed. I already try it.

Plz look a #1098

Comment (GuilhemN) [reply .travis.yml]: That's not a problem, we do not solve issues by silencing them. The CI will be fixed soon enough but this PR is unrelated.

Comment (matrix818181) [reply .travis.yml]: Done !

**Comment by GuilhemN** [comment]
Adding some tests to ensure this works fine would be great :)

**Comment by GuilhemN** [comment]
Closing as I just merged #1126. Thanks!
